libs/commons/File_type.ml

```diff
@@ -70,6 +70,7 @@ and pl_type =
   | Erlang
   | Go
   | Rust
+  | Move_on_sui
```

Comment:
I think we probably just want to have | Move here
and remove Move_on_xxx and have a single Move.